### ISSUE TITLE
XSLTFilter namePattern failures can show wrong name in error message

### ISSUE THREAD

**Issue by stroomdev10** [gchq/stroom#3570]
Appears to report the name associated with the UUID at the time the pipeline was set up. (its stored in the Pipeline XML)

Would be better if it resolved the name from the UUID instead

**Comment by at055612** [comment]
Now resolves the name based on UUID and displays the UUID for clarity
```
WARN: (Code pane) - [?:?:?] No XSLT found with name 'TEST_REFERENCE_DATA-EVENTS_XXXz' from pattern '${pipeline}z' - using default XSLT 'XSLT 2' (61588fcf-bf81-418c-a421-92bc834587c5)

WARN: (Code pane) - [?:?:?] Found 2 XSLTs with name 'XSLT 2' from pattern 'XSLT 2' - using XSLT 'XSLT 2' (61588fcf-bf81-418c-a421-92bc834587c5). All matching UUIDs: 61588fcf-bf81-418c-a421-92bc834587c5, 9cdcffa8-f4e9-4b65-9baf-9fa5d517f28e)

FATAL: (Code pane) - [?:?:?] XSLT 'XSLT 2' (61588fcf-bf81-418c-a421-92bc834587c5) appears to have been deleted
WARN: (Code pane) - [?:?:?] No XSLT found with name 'XSLT 2' from pattern 'XSLT 2' - no default XSLT specified
```

**Comment by at055612** [comment]
Fixed in >7.2-beta.42
